client.go

```diff
@@ -228,7 +228,7 @@ func startContainer(img string, cmd []string, streaming, wait bool) (string, err
 			}
 		}()
 		go func() {
-			if _, err := io.Copy(os.Stdout, rc); err != nil {
+			if _, err := io.Copy(dst, rc); err != nil {
```

Comment:
I saw that some pieces of the code are sending a nil `dst`, what is going to happen here?